src/development/python_package_release_procedure.rst

```diff
@@ -0,0 +1,138 @@
+.. _Semver: https://semver.org
+
+==================================
+Software Package Release Procedure
+==================================
+
+Versioning Procedure
+--------------------
+
+As part of our goal to align all developmental efforts to one standard, we
+have documented a procedure of how we would like all the *SKA* developers to
+version their releases and what process to follow in ensuring that they are 
+able to make use of the existing Gitlab CI/CD pipeline to automate the building
+of python packages, for now, and have them published on the *SKA* pypi registry
+which is hosted on *Nexus*.
+
+
+Versioning scheme to use
+========================
+
+The scheme chosen to be adopted by the *SKA* developer community is the semantic versioning
+scheme.
+More information regarding this scheme can be found on the Semver_ site.
+
+How to mark a release
+=====================
+
+A developer should make use of the git annotated tags to indicate that this
+current commit is to serve as a release. For example:
+
+.. code:: bash
+
+  $ git tag -a "1.0.0" -m "Release 1.0.0. This is a patch release that resolves
+    issue <JIRA issue>."
+
+After that is complete, then the tag needs to be published to the origin:
+
+.. code:: bash
+
+  $ git push origin <tag_name>
+
+
+Packaging Procedure
+-------------------
+
```

Comment:
Can we have a sub heading here: `Python`
Since other languages may have other instructions.

Reply:
I added a commit for this.